sematic/resolvers/worker.py

```diff
@@ -169,12 +175,24 @@ def main(run_id: str, resolve: bool):
         raise e
 
 
-if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO)
+def _create_log_file_path(file_name: str) -> str:
+    temp_dir = tempfile.gettempdir()
+    sematic_temp_logs_dir = pathlib.Path(temp_dir) / pathlib.Path("sematic_logs")
+    if not pathlib.Path(sematic_temp_logs_dir).exists():
+        os.mkdir(sematic_temp_logs_dir)
+    return (pathlib.Path(sematic_temp_logs_dir) / pathlib.Path(file_name)).as_posix()
 
+
+if __name__ == "__main__":
+    print("Starting Sematic Worker")
     args = parse_args()
+    log_kind = "resolve" if args.resolve else "calculation"
+    log_prefix = f"logs/run_id/{args.run_id}/{log_kind}"
+    path = _create_log_file_path("worker.log")
 
-    logger.info("Worker CLI args: run_id=%s", args.run_id)
-    logger.info("Worker CLI args:  resolve=%s", args.resolve)
+    with ingested_logs(path, LOG_UPLOAD_INTERVAL_SECONDS, log_prefix):
```

Comment:
It makes testing easier. In a test, you will want to be able to have a much shorter upload interval. I can make it so that there's a default set in `log_streamer` though